engines/instance_verification/lib/instance_verification/engine.rb

```diff
@@ -1,4 +1,17 @@
+require 'net/http'
+
+ACTIVATE_PRODUCT_URL = 'https://scc.suse.com/connect/systems/products'.freeze
+SYSTEM_SUBSCRIPTION_URL = 'https://scc.suse.com/connect/systems/subscriptions'.freeze
+SUBSCRIPTIONS_PRODUCTS_URL = 'https://scc.suse.com/connect/subscriptions/products'.freeze
```

Comment:
Why does instance_verification need to access SCC? I thought it's checking with the instance data, if the client system is allowed to use a specific product?